sync-repos.sh

```diff
@@ -2,24 +2,27 @@ set -ex
 
 pip3 install -U pip
 
-cd /opt/genienlp/
-GENIENLP_HEAD=`git rev-parse HEAD`
-if [ -n "${GENIENLP_VERSION}" ] && [ "${GENIENLP_VERSION}" != "${GENIENLP_HEAD}" ]; then
-  git fetch
-  git checkout ${GENIENLP_VERSION}
-  pip3 install -e .
-fi
-
 if [ -d /opt/bootleg/ ] ; then
   cd /opt/bootleg/
   BOOTLEG_HEAD=`git rev-parse HEAD`
   if [ -n "${BOOTLEG_VERSION}" ] && [ "${BOOTLEG_VERSION}" != "${BOOTLEG_HEAD}" ]; then
     git fetch
     git checkout ${BOOTLEG_VERSION}
     pip3 install -r requirements.txt && pip3 install -e .
+    python3 -m spacy download en_core_web_sm
   fi
 fi
 
+
+cd /opt/genienlp/
+GENIENLP_HEAD=`git rev-parse HEAD`
+if [ -n "${GENIENLP_VERSION}" ] && [ "${GENIENLP_VERSION}" != "${GENIENLP_HEAD}" ]; then
+  git fetch
+  git checkout ${GENIENLP_VERSION}
+  pip3 install -e .
+fi
```

Comment:
Is this change in order right? Pip installing genienlp will uninstall bootleg, and override any bootleg version you specify.

Reply:
Before the order mattered cause bootleg pinned all package versions. I relaxed this so genienlp and bootleg agree on shared packages. I can still change the order to match the dockerfile.

Reply:
In fact, let me do a test.